### ISSUE TITLE
Load orders for files inside archives.

### ISSUE THREAD

**Issue by CortexReaver** [nstlaurent/DoomLauncher#254]

**Comment by CortexReaver** [comment]
Correspondingly with my previous mock-ups, here is what I proposing for selecting files inside archives:

![31-05-2022 094042](https://user-images.githubusercontent.com/34350690/171238392-651927d7-7aa7-4f46-a165-b07f6c4dcd7d.png)

**Comment by marxistguy** [comment]
seriously this, i need this feature to be added, its a hassle to sort the files individually with its archive and/or utilities, and for the love of me I can not find the option so far to sort the wads and pk3s inside the archives itself

**Comment by mikew** [comment]
Agreed, it doesn't need to be complicated either (like the second screenshot). There's already a UI included that shows the user all the files. If we could arrange that list things would be great.

**Comment by CortexReaver** [comment]
> Agreed, it doesn't need to be complicated either (like the second screenshot). There's already a UI included that shows the user all the files. If we could arrange that list things would be great.

The idea was to make profiles, which includes multiple archives that you can quickly add to any other mod. Colors shows you to what profiles they are corresponding to, so you know what are you about to delete.
But yes, without profiles it might be simplified, although I think it is a good thing to see from which archive the particular file is coming. So in that case you might need to show the hierarchy every time you separate files from the same archive.

![171238392-651927d7-7aa7-4f46-a165-b07f6c4dcd7d](https://github.com/nstlaurent/DoomLauncher/assets/34350690/4c70573b-fa31-440f-afa8-3b60ccd10ff2)
